src/App.js

```diff
@@ -21,94 +21,91 @@ import NoFavorites from './component/NoFavorites/NoFavorites';
 import Donate from './component/Donate/Donate';
 
 function ProtectedRoute({ isAuthenticated, children }) {
-    if (!isAuthenticated) {
```

Comment:
Was the spacing formatted by you or your IDE(VS code)? Might be worth looking into having the entire team have the same formatting settings in this project to avoid different formatting applied by each team members IDE.

Reply:
https://prettier.io/docs/en/options.html

Reply:
By Prettier in IDE(VS code). 
>Might be worth looking into having the entire team have the same formatting setting
Good point, but we didn't do that before, so we don't know how to do that. To figure it out ourselves could take a lot of time. Could you meet with all of as and show as haw it could be done? I can schedule a meeting.